pkg/deploy/api/v1beta3/conversion_test.go

```diff
@@ -206,7 +206,7 @@ func newInt64(val int64) *int64 {
 	return &val
 }
 
-func newInt(val int) *int {
+func newInt32(val int32) *int32 {
```

Comment:
`newInt32Ptr` ? I don't think this is returning a new int32, just a pointer to existing one.